app/lib/hooks/trigger_repeat.rb

```diff
@@ -0,0 +1,34 @@
+module Hooks
+  class TriggerRepeat < ::Dynflow::ExecutionPlan::Hooks::Abstract
```

Comment:
Where is this hook registered?

Reply:
I forgot to open a PR for it... https://github.com/adamruzicka/foreman_remote_execution/commit/3bb154d74e5fc42e1154c9c818766acabcbd7a87